### DISCUSSION TITLE
Where are the channels ? Here

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#413] [Q&A] [answered]
I am not sure which is which but does you guys have those lists ? I cannot find them on your m3u8 lists, thought. 

24-7 RETRO TV
RETRO CRUSH
THIS TV NETWORK
WEATHERSPY

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
